src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+import { PreparedTodo } from '../../types/Todo';
+
+type Props = {
+  todosList: PreparedTodo[]
+};
+
+export const TodoList: React.FC<Props> = ({ todosList }) => (
+  <ul className="content">
+    {todosList.map(todo => (
+      <li key={todo.id} className="">
```

Comment:
do not leave empty attributes

```suggestion
      <li key={todo.id}>
```